client/components/Slide/Components/ResponseRecall/Display.jsx

```diff
@@ -60,14 +62,27 @@ class Display extends React.Component {
                 : 'Loading your previous response'
             : 'Participant response will appear here';
 
-        return content.endsWith('mp3') ? (
+        return (
             <Message
+                floating
+                style={{ whiteSpace: 'pre-wrap' }}
                 content={
-                    <audio src={`/api/media/${content}`} controls="controls" />
+                    content.endsWith('mp3') ? (
+                        <React.Fragment>
+                            <audio
+                                src={`/api/media/${content}`}
+                                controls="controls"
+                            />
+                            <blockquote className="audiotranscript__blockquote">
+                                {response.response.transcript ||
+                                    '(Transcribing, please hold)'}
```

Comment:
I was a little confused by this message to please hold as I thought the transcription would happen in like a second or two rather than a minute (which is what I experienced). What do you think of letting the user know that this may take a minute or two?

Reply:
Sure!